app/javascript/components/patient/Patient.js

```diff
@@ -151,12 +151,11 @@ class Patient extends React.Component {
             />
           )}
           <Col sm={12}>
-            <h3>
-              <span aria-label={formatName(this.props.details)} className="pr-2">
-                {formatName(this.props.details)}
-              </span>
+            <h2 className="secondary-title">
```

Comment:
It seems to me that given the nature of react and the inherent re-usability of components, statically defining any header tags is dangerous. In this case, we know at the moment that the `Patient` component happens to only be used in places where we would want its top level headers to be `h2`, but there's nothing guaranteeing that, and I'm sure if we propagate usage of header tags throughout other components, we'll run into situations where the top level within a component is variable.

How would you feel about making this dynamic by parameterizing the top header level for a component in the props? I was hoping there might be a more built-in react way to do this, but after briefly googling, I found this [blog post](https://dev.to/s_aitchison/psa-stop-hard-coding-heading-levels-in-your-react-components-2ekp) and this [stack overflow post](https://stackoverflow.com/a/33471928/3381648), which seem to agree.

The [`react-headings`](https://www.npmjs.com/package/react-headings) package does exactly what I was hoping for, but it doesn't have enough downloads for me to be comfortable using it.

Reply:
fixed using different custom solution